sdbus-c++ compiler warnings

```
In file included from /mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/common/sdbus/third_party/sdbus-cpp/src/VTableUtils.c:27:
/mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/common/sdbus/third_party/sdbus-cpp/src/VTableUtils.h:57:39: warning: a function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes]
   57 | sd_bus_vtable createSdBusVTableEndItem();
      |                                       ^
      |                                        void
/mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/common/sdbus/third_party/sdbus-cpp/src/VTableUtils.c:101:39: warning: a function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes]
  101 | sd_bus_vtable createSdBusVTableEndItem()
      |                                       ^
      |                                        void
2 warnings generated.
[9/12] Building CXX object plugins/common/sdbus/third_party/sdbus-cpp/CMakeFiles/sdbus-c++-objlib.dir/src/Connection.cpp.o
/mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/common/sdbus/third_party/sdbus-cpp/src/Connection.cpp:420:75: warning: implicit conversion changes signedness: 'short' to 'uint32_t' (aka 'unsigned int') [-Wsign-conversion]
  420 |     auto r = sd_event_source_set_io_events(sdIoEventSource, sdbusPollData.events);
      |              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                  ~~~~~~~~~~~~~~^~~~~~
1 warning generated.
```

--- a/plugins/common/CMakeLists.txt
+++ b/plugins/common/CMakeLists.txt
@@ -8,6 +8,8 @@ add_library(plugin_common STATIC
         uuid/uuidxx.cc
 )
 add_subdirectory(sdbus)
+#  warning suppression
+target_compile_options(sdbus-c++-objlib PRIVATE "-Wno-strict-prototypes" "-Wno-sign-conversion")
 
 target_include_directories(plugin_common PUBLIC . ${PROJECT_BINARY_DIR})
 target_compile_definitions(plugin_common PUBLIC EGL_NO_X11)

